app/controllers/registration/otherbusinessinvolvements/OtherBusinessActivelyTradingController.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.registration.otherbusinessinvolvements
+
+import config.{AuthClientConnector, BaseControllerComponents, FrontendAppConfig}
+import controllers.BaseController
+import controllers.registration.otherbusinessinvolvements.{routes => obiRoutes}
+import forms.otherbusinessinvolvements.OtherBusinessActivelyTradingForm
+import models.OtherBusinessInvolvement
+import play.api.mvc.{Action, AnyContent}
+import services.OtherBusinessInvolvementsService.StillTradingAnswer
+import services.{OtherBusinessInvolvementsService, SessionProfile, SessionService}
+import views.html.otherbusinessinvolvements.OtherBusinessActivelyTradingView
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class OtherBusinessActivelyTradingController @Inject()(val sessionService: SessionService,
+                                                       val authConnector: AuthClientConnector,
+                                                       val otherBusinessInvolvementsService: OtherBusinessInvolvementsService,
+                                                       view: OtherBusinessActivelyTradingView)
+                                                      (implicit appConfig: FrontendAppConfig,
+                                                       val executionContext: ExecutionContext,
+                                                       baseControllerComponents: BaseControllerComponents)
+  extends BaseController with ObiIndexValidation with SessionProfile {
+
+  def show(index: Int): Action[AnyContent] = isAuthenticatedWithProfile() {
+    implicit request =>
+      implicit profile =>
+        validateIndex(index, routes.OtherBusinessActivelyTradingController.show) {
+          otherBusinessInvolvementsService.getOtherBusinessInvolvement(index).flatMap {
+            case Some(OtherBusinessInvolvement(_, _, _, _, _, Some(true))) =>
```

Comment:
I'd change this to `case Some(OtherBusinessInvolvement(_, _, _, _, _, activelyTrading))` then on line 48 you can fill the form with the answer using fold. e.g. `activelyTrading.fold(OtherBusinessActivelyTradingForm.form)(OtherBusinessActivelyTradingForm.fill)`